tck/README.md

```diff
@@ -38,8 +38,9 @@ Since the TCK is aimed at Reactive Stream implementers, looking into the sources
 and should help during a libraries implementation cycle.
 
 In order to make mapping between test cases and Specification rules easier, each test case covering a specific
-Specification rule abides the following naming convention: `spec###_DESC` where:
+Specification rule abides the following naming convention: `TYPE_spec###_DESC` where:
 
+* `TYPE` is one of: `required`, `optional`, `stochastic` or `untested` which describe if this test is covering a Rule that MUST or SHOULD be implemented. The specific words are explained in detail below.
```

Comment:
`The specific words are explained in detail below.` <- where below (i.e. it is not quick to find)